app/models/permissions/scientist_permission.rb

```diff
@@ -1,19 +1,18 @@
 # frozen_string_literal: true
 
 ##
-# Permissions for a Standard User
-# Allowed to create a scan in the user interface and the api.
+# Permissions for a Scientist
+#
 module Permissions
   class ScientistPermission < BasePermission
     def initialize(user)
       super
       allow :scans, [:create]
+      allow :upload_labware, [:create]
+      # Below is used from Sequencescape labware reception
```

Comment:
The api is available for any external app. It could be used by others but labware reception is an example.